### PR TITLE
Noarch package

### LINKED ISSUE #34: @conda-forge-admin, please add noarch: python
### Additional comment:

This package does not have any architectural dependencies per se and can be noarch (see https://github.com/MDAnalysis/mdanalysis/issues/3727 and manual attempt PR #33)

### PR DESCRIPTION
**Pull request by orbeckst**
Change mdanalysistests into a noarch package (see https://github.com/MDAnalysis/mdanalysis/issues/3727)

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by orbeckst** [comment]
@conda-forge-admin, please rerender

**Comment by orbeckst** [comment]
Comments from PR #32 
- @IAlibay : I believe the way forward in meta.yml is to have noarch in the build section: https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python

File: recipe/meta.yaml [hunk 1]
```diff
@@ -12,16 +12,14 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 0
+  number: 1
+  noarch: python
   script: python setup.py install --single-version-externally-managed --record record.txt
```
Comment (orbeckst) [inline recipe/meta.yaml]: Can this be rewritten as
```
script: {{ PYTHON }} -m pip install . --no-deps -vv
```

(possibly change requirements.host setuptools --> pip)?

**Comment by orbeckst** [comment]
@conda-forge-admin, please rerender

**Comment by orbeckst** [comment]
I'll probably close my manual attempt here and let conda-smithy do everything in PR #34 ...

File: recipe/meta.yaml (hunk 1 above)
Comment (orbeckst) [reply recipe/meta.yaml]: no-deps will not work as we need MDA.

Comment (IAlibay) [reply recipe/meta.yaml]: Isn't that already defined under `run`?

**Comment by orbeckst** [comment]
The automated PR #35 really only added noarch but there was other stuff to fix. It looks as if this PR is just as good (and already has more things removed).

@IAlibay perhaps you could have a quick look when you have time?

**Comment by IAlibay** [comment]
> The automated PR #35 really only added noarch but there was other stuff to fix. It looks as if this PR is just as good (and already has more things removed).
> 
> @IAlibay perhaps you could have a quick look when you have time?

👍🏽 I'll give it a look through in the morning.

File: recipe/meta.yaml (hunk 1 above)
Comment (orbeckst) [reply recipe/meta.yaml]: Ah, you're right... Do we need the `--single-version-externally-managed --record record.txt` options for something??

Comment (orbeckst) [reply recipe/meta.yaml]: I changed it to use pip.

**Review by IAlibay — APPROVED** [review]
This seems fine to me, I don't understand thing enough to be 100% sure, but the best thing we can do is test it out and see what happens.

**Pull request merged**